src/sentry/static/sentry/app/utils/performance/quickTrace/utils.ts

```diff
@@ -16,6 +17,22 @@ export function isTransaction(event: Event): event is EventTransaction {
   return event.type === 'transaction';
 }
 
+/**
+ * An event can be an error or a transaction. We need to check whether the current
+ * event id is in the list of errors as well
+ */
+function isCurrentEvent(
+  event: TraceFull | QuickTraceEvent,
+  currentEvent: Event
+): boolean {
+  return (
+    event.event_id === currentEvent.id ||
+    (!isTransaction(currentEvent) &&
+      event.errors !== undefined &&
+      event.errors.filter(e => e.event_id === currentEvent.id).length > 0)
```

Comment:
🏌️
```javascript
!isTransaction(currentEvent) && event.errors?.some(e => e.event_id === currentEvent.id)
```

Reply:
Didn't end up doing this cause of weird ts errors lol. But cleaned up the boolean by splitting into 2 ifs instead